tests/test_colors.py

```diff
@@ -136,7 +136,8 @@ def test_py3():
         return
     value = Color('{red}this is a test.{/red}')
 
-    assert '\033[31mss\033[39m' == Color('{red}ß{/red}').casefold()
+    if hasattr(str, 'casefold'):
```

Comment:
I think it's better if you do hasattr(Color, 'casefold') since Color subclasses unicode in py2. Even though this condition will work, it's more consistent for tests to run against the same builtin type (which Color subclasses so it handles choosing the right type anyways).